InfrastructureManager/infrastructure_manager.py

```diff
@@ -110,16 +110,18 @@ def describe_instances(self, parameters, secret):
       a valid 'reservation_id' parameter, this method will return a
       dictionary containing information regarding the requested past
       virtual machine deployment. This returned map contains several
-      keys including 'success', 'state', 'reason' and 'vm_info'. The value
-      of 'success' could be True of False depending on the outcome of the
-      virtual machine deployment process. If the value of 'success' happens
-      to be False, the 'reason' key would contain more details as to what
-      caused the deployment to fail. The 'state' key could contain a 'pending'
-      value or a 'running' value depending on the current state of the
-      virtual machine deployment. And finally the 'vm_info' key would point
-      to a another dictionary containing the IP addresses of the spawned virtual
-      machines. If the virtual machine deployment had failed or still in the
-      'pending' state, this key would contain the value None.
+      keys including 'success', 'state', 'reason' and 'vm_info' (for run
```

Comment:
@obino I meant for when it is trying to describe a reservation that was returned by run_instances. The "reservation" is within a dictionary maintained by the IAAS and it gets updated with the state of running the agent call. The main change in this docstring is just that you can call describe instances on a terminate_instances reservation, but that won't return vm_info because it doesn't need to.


if that makes any sense

Reply:
yes it does. Perhaps add an 'only'? like for run instances reservations only?

Reply:
would it help if I explicitly said what was returned for a run_instances reservation and what is returned for a terminate_instances reservation?